site/contents/write_ups/2021/11-08-21_writeup.md

```diff
@@ -0,0 +1,119 @@
+
+That being said, the actions of some will make more of a difference than others. The classic: "Everyone stop rinsing dishes before putting dishes in the dishwasher because that'll stop climate change" is a nice idea but fails to acknowledge the comparative cost of [launching billionaires into space](https://www.independent.co.uk/climate-change/news/bezos-rocket-carbon-emissions-blue-origin-b1888059.html). Grassroots change and influence will matter, but that is likely to focus on small singular issues. Focus on the small may not get us where we need to be, nor nearly quick enough.
+
+Fixing climate change is a bit like a terrible group project. Seeing larger contributers slack off can make it tempting to throw our hands up in defeat and resign ourselves to our fate. This same cynicism is seen for example among people who choose not to vote, asking "what difference can one person make?" Of course every vote counts (even though in some systems, some votes count more than others), but it leads to a question of what can any single person do about it?
+
+## What can we do about it?
+
+The group was split by this question. 
+
+On the one hand, some had the belief that individual action won't make any difference. Although collectively we might be able to do something. Actions like being vegetarian that many people take up could be easily offset by the single action of one individual or organisation (e.g. shooting billionaires into space). 
+
+Making individual efforts also helps to offset the doom. Democracy i a solution that we can all contribute to (even though it feels like it doesn't make a difference). Voting for politicians with green policies is something we can contribute to, but we might never be able to offset the lobbying by larger cooperations. 
+
+The media also platform anti-climate change "scientists" and spokespeople. For example [Boris Johnson's father being interviewed as an environmentalist by the BBC](https://www.thenational.scot/news/19504423.bbc-newsnight-greens-appalled-stanley-johnsons-appearance/) - there is inherent bias in the media. 
+
+This idea of "equal debate" is flawed. If 10% of people believe in one thing and 90% don't, giving representives from both viewpoints equal airtime is not "equal". This has been bought to light during the pandemic, where we see qualified epidemeologists debating youtubers.
+
+What constitutes an expert? It's usually seen as negative when people change their minds, but acually that's a reflection of changing evidence (i.e. science). An interesting exploration on this tangent can be found in this [video](https://www.youtube.com/watch?v=LRNkDZy30xU).
+
+On the other hand, we can be more positive about our ability to affect change. 
```

Comment:
```suggestion
On the other hand, a movement will take the actions of all of us, so its still important to make individual actions where we can, rather than feeling defeatist about our ability to affect change. 
```